src/utils/LaunchUtils.ts

```diff
@@ -64,3 +66,9 @@ export const throwIfNoGameDir = async (game: Game): Promise<void> => {
         throw error;
     }
 };
+
+export async function isProtonRequired(activeGame: Game) {
+    return activeGame.activePlatform.storePlatform === Platform.STEAM
```

Comment:
I assume this intentionally doesn't apply to STEAM_DIRECT games?

Reply:
Good spot, have added.